agent/run.go

```diff
@@ -14,6 +14,12 @@ const (
 	TaskTypeSink TaskType = "sink"
 )
 
+type PluginRun struct {
```

Comment:
let's not create a struct here if it is only for metric gathering purposes only.